paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/DefaultConfirmationHandlerTest.kt

```diff
@@ -670,9 +675,9 @@ class DefaultConfirmationHandlerTest {
             ),
         shouldRegister: Boolean = true,
         savedStateHandle: SavedStateHandle = SavedStateHandle(),
-        dispatcher: CoroutineDispatcher = UnconfinedTestDispatcher(),
+        dispatcher: CoroutineDispatcher = StandardTestDispatcher(),
```

Comment:
Had to move a bunch of these over to `StandardTestDispatcher`. Otherwise we got failures (including the link one I had @samer-stripe look into before I opened this up for review).